app/models/solidus/gateway/braintree_gateway.rb

```diff
@@ -7,6 +7,7 @@ class Gateway::BraintreeGateway < ::Spree::Gateway
     preference :public_key, :string
     preference :private_key, :string
     preference :always_send_bill_address, :boolean, default: false
+    preference :transmit_shipping_address, :boolean, default: true
```

Comment:
I didn't want to change the naming scheme of the old config. My hope is that we'll give this a big ol' refactor at some point and make this all make a whole lot more sense.